content/providers/01-ai-sdk-providers/08-amazon-bedrock.mdx

```diff
@@ -442,6 +442,27 @@ console.log(text); // text response
 See [AI SDK UI: Chatbot](/docs/ai-sdk-ui/chatbot#reasoning) for more details
 on how to integrate reasoning into your chatbot.
 
+## Extended Context Window
+
+Claude Sonnet 4 models on Amazon Bedrock support an extended context window of up to 1 million tokens when using the `context-1m-2025-08-07` beta feature.
+
+```ts
+import { bedrock } from '@ai-sdk/amazon-bedrock';
+import { generateText } from 'ai';
+
+const result = await generateText({
+  model: bedrock('us.anthropic.claude-sonnet-4-20250514-v1:0'),
+  prompt: 'analyze this large document...',
+  providerOptions: {
+    bedrock: {
+      additionalModelRequestFields: {
+        anthropic_beta: ['context-1m-2025-08-07'],
```

Comment:
should be `anthropicBeta`, compare `budgetTokens`. We use camelCase in all our options and then map it to under_score in the request body as needed.